examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -218,6 +213,20 @@ export class TaskList extends DataObject implements ITaskList {
      * DataObject, by registering an event listener for changes to the task list.
      */
     protected async hasInitialized(): Promise<void> {
+        const saved = this.root.get<IFluidHandle<SharedMap>>("savedData");
+        if (!saved) {
+            throw new Error("savedData was not initialized");
+        } else {
+            this._savedData = await saved.get();
+        }
```

Comment:
Stylistic nit:  Since the conditional throws and halts execution of this method, we can leave out the else (same below for the draftData)
```suggestion
        }
        this._savedData = await saved.get();
```